src/vtools-analysis.py

```diff
@@ -539,6 +550,15 @@ def get_options(argv):
         metavar="output-file",
         help="output file",
     )
+    parser.add_argument(
```

Comment:
add "--no-dump-audio-info" option

Reply:
by default the audio info is not dumped. 
only when --dump-audio-info is specified, audio info are dumped to csv.
do I need to add "--no-dump-audio-info"?

Reply:
yes. It will make changing the default just a one liner. We do this for all the other Boolean settings